l1-contracts-foundry/script-config/config-deploy-l1.toml

```diff
@@ -0,0 +1,27 @@
+gas_price = 0
+era_chain_id = 9
```

Comment:
Just to make sure, on different Era instances ( stage testnet mainnet) will have different chainIds. This will have to be considered

Reply:
yes, for sure. 

this configs no more than example